deltachat-rpc-client/src/deltachat_rpc_client/_utils.py

```diff
@@ -47,36 +47,33 @@ def __setattr__(self, attr, val):
 def run_client_cli(
     hooks: Optional[Iterable[Tuple[Callable, Union[type, "EventFilter"]]]] = None,
     argv: Optional[list] = None,
-    **kwargs,
 ) -> None:
     """Run a simple command line app, using the given hooks.
 
     Extra keyword arguments are passed to the internal Rpc object.
     """
     from .client import Client
 
-    _run_cli(Client, hooks, argv, **kwargs)
+    _run_cli(Client, hooks, argv)
 
 
 def run_bot_cli(
     hooks: Optional[Iterable[Tuple[Callable, Union[type, "EventFilter"]]]] = None,
     argv: Optional[list] = None,
-    **kwargs,
 ) -> None:
     """Run a simple bot command line using the given hooks.
 
     Extra keyword arguments are passed to the internal Rpc object.
     """
     from .client import Bot
 
-    _run_cli(Bot, hooks, argv, **kwargs)
```

Comment:
Don't know if it is actually unused e.g. for `rpc_server_path`, at least the documentation should be updated.

Reply:
the reason `**kwargs` existed in the past was to be able to control the created process, ex. IIRC TUI clients need to tweak it so stderr don't dirty the screen,

I am talking about the removal in the Rpc class, but other approaches like allowing to pass a Popen directly to the start method in RpcProcess would also work I guess